src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -452,4 +485,18 @@ $time-slider-color-has-data: color.adjust($primary, $lightness: 30%);
         text-align: center;
     }
 }
+.time-slider-bar-cursor-year {
+    &.form-control {
```

Comment:
@ltshb I opted to only show a red border in case of an invalid input since the space is very limited, should there be something more like a popup?

Reply:
You could simply add a tippy like this

```diff
diff --git a/src/modules/map/components/toolbox/TimeSlider.vue b/src/modules/map/components/toolbox/TimeSlider.vue
index 6321cc9c..e13a8875 100644
--- a/src/modules/map/components/toolbox/TimeSlider.vue
+++ b/src/modules/map/components/toolbox/TimeSlider.vue
@@ -4,9 +4,15 @@ import { computed, onMounted, onUnmounted, ref, watch } from 'vue'
 import { useStore } from 'vuex'
 
 import { round } from '@/utils/numberUtils'
+import { useTippyTooltip } from '@/utils/useTippyTooltip'
 
 const dispatcher = { dispatcher: 'TimeSlider.vue' }
 
+const { refreshTippyAttachment } = useTippyTooltip(
+    '.time-slider-bar-cursor-year[data-tippy-content]',
+    { theme: 'danger' }
+)
+
 /**
  * The oldest year in our system is from the layer Journey Through Time (ch.swisstopo.zeitreihen)
  * which has data from the year 1844
@@ -284,6 +290,8 @@ function setYearToInputIfValid() {
         allYears.value.includes(parseInt(displayedYear.value))
     ) {
         setCurrentYearAndDispatchToStore(parseInt(displayedYear.value))
+    } else {
+        refreshTippyAttachment()
     }
 }
 </script>
@@ -316,6 +324,7 @@ function setYearToInputIfValid() {
                         v-model="displayedYear"
                         class="form-control time-slider-bar-cursor-year"
                         :class="{ 'is-invalid': invalidYear }"
+                        :data-tippy-content="invalidYear ? 'invalid_slider_year' : null"
                         data-cy="time-slider-bar-cursor-year"
                         maxlength="4"
                         type="text"
```

You need to create a new translation key for the invalid year, the content should be kind of: `Year should be between $X and $Y` and you could then replace X and Y by the min max year configured.

Reply:
This will probably look nicer than using the form-control native tooltip